bitcoin/src/blockdata/opcodes.rs

```diff
@@ -687,6 +692,7 @@ impl All {
             (OP_VERIF, _) | (OP_VERNOTIF, _) | (OP_INVALIDOPCODE, _) => Class::IllegalOp,
 
             // 15 opcodes illegal in Legacy context
+            #[rustfmt::skip]
```

Comment:
You cheated, @tcharding!

Reply:
As in this probably belongs in a separate commit to 0c95bb7b66bc5de3d4b379bd5aaf727fe1b18bae.

Reply:
Bollocks! You got me. However, review for the win. Will fix and re-spin ... and do `psbt`.